src/server/interpreter.py

```diff
@@ -13,91 +13,61 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import random
-import glob
 
-from PIL import Image
 from cmd import Cmd
 
-from src.parser.parser import Parser
-
 
 class EvaCommandInterpreter(Cmd):
 
     # Store results from server
     _server_result = None
+    url = None
+
+    def __init__(self):
+        super().__init__()
+
+        # Create table on connecting to server
 
     def set_protocol(self, protocol):
         self.protocol = protocol
 
     def do_greet(self, line):
         print("greeting")
 
-    def onecmd(self, s):
+    def emptyline(self):
+        print("Enter a valid query.")
```

Comment:
Do the print statements in this file display on the client's prompt? @jarulraj Shouldn't we do something like self.stdout.write()?